src/App.tsx

```diff
@@ -15,36 +16,111 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+enum SortType {
+  Alphaberticaly = 'Sort alphabetically',
+  Length = 'Sort by length',
+  Reverse = 'Reverse',
+}
+
+const SORT_FIELD_ALPHABET: SortType = SortType.Alphaberticaly;
+const SORT_FIELD_LENGTH: SortType = SortType.Length;
+
+function getPreparedGoods(
+  goods: string[],
+  { sortField }: { sortField: SortType },
+): string[] {
+  const preparedGoods = [...goods];
+
+  if (sortField) {
+    switch (sortField) {
+      case SORT_FIELD_ALPHABET:
+        preparedGoods.sort((a, b) => a.localeCompare(b));
+        break;
+      case SORT_FIELD_LENGTH:
+        preparedGoods.sort((a, b) => a.length - b.length);
+        break;
+      default:
+        break;
+    }
+  }
```

Comment:
There is no need to check `if (sortField)` before the switch statement because `sortField` is guaranteed to be of type `SortType` and cannot be `null` or `undefined`. You can safely remove the `if` condition.